### ISSUE TITLE
Say "DreamGL" in README.md as SEO

### ISSUE THREAD

**Issue by mncharity** [dreemproject/dreemgl#107]
User experience report: I attempted to point someone at DreemGL.  But I only remembered the name's sound, "dream gl", not its spelling.  Googling for "dreamgl webgl" and similar, was unsuccessful.

Someday google may suggest "Did you mean DreemGL?".  But for now...

Perhaps add the word DreamGL to README.md to improve searchability.
Perhaps as "... DreemGL (sounds like DreamGL) ...", or "not DreamGL".

**Comment by raju-bitter** [comment]
Good idea, thanks for reporting your experience. We had a long discussion around the naming and the confusion which might be caused by the modified spelling of "dream". SEO optimization is certainly needed, will look into that next week. Thanks again!
